src/router.js

```diff
@@ -8,6 +8,23 @@ const messages = {
     message: 'Email ou mot de passe incorrect.',
   },
 };
+const DEFAULT_EIDAS_LEVEL = 'eidas3';
+
+export const getEidasLevel = (req) => {
+  if (!req.session || !req.session.infos || !req.session.infos.acr_values) {
+    return DEFAULT_EIDAS_LEVEL;
+  }
+
+  const validEidasLevel = ['eidas1', 'eidas2', 'eidas3'];
```

Comment:
This constant could be defined outside of the function, and in caps, like  `DEFAULT_EIDAS_LEVEL` is;

Those two constant might also be stored somewhere else in the project, maybe in a configuration file to allow easy tweak of this parameter (I'm not aware of a real world use case, though).